redbot/setup.py

```diff
@@ -166,7 +166,9 @@ def basic_setup():
     print()
     print(
         "Your basic configuration has been saved. Please run `redbot <name>` to"
-        " continue your setup process and to run the bot."
+        " continue your setup process and to run the bot.\n\n"
+        "First time? Read the quickstart guide:\n"
+        "https://docs.discord.red/en/latest/getting_started.html"
```

Comment:
This should point to stable. I'm aware the link doesn't exist currently, but we don't generally link users to dev changes as we don't recommend the development version for normal use, and this is occurring with a message to first time users, and development users can deal with a known reason why a link may not exist yet.

Reply:
Will do.